src/sidebars/favourites/tearout.js

```diff
@@ -0,0 +1,218 @@
+(function(window) {
+    'use strict';
+
+    angular.module('openfin.tearout', ['openfin.geometry', 'openfin.hover', 'openfin.store'])
+        .directive('tearable', ['geometryService', 'hoverService', 'storeService', function(geometryService, hoverService, storeService) {
+            return {
+                restrict: 'C',
+                link: function(scope, element, attrs) {
+                    var tileWidth = 230,
+                        tileHeight = 100;
+
+                    function createConfig() {
+                        return {
+                            'name': scope.stock.code + ' window',
+                            'maxWidth': tileWidth,
+                            'minHeight': tileHeight,
+                            'defaultWidth': tileWidth,
+                            'defaultHeight': tileHeight,
+                            'width': tileWidth,
+                            'height': tileHeight,
+                            'autoShow': false,
+                            'url': 'sidebars/favourites/tearout.html',
+                            'frame': false,
+                            'resizable': false,
+                            'maximizable': false
+                        };
+                    }
+
+                    var tearoutWindow = new fin.desktop.Window(createConfig());
+
+                    // TODO: Improve this. Search for first class element upwards?
+                    var dragElement = element[0];
+                    var tearElement = dragElement.parentNode.parentNode;
+
+                    fin.desktop.main(function() {
+                        function initialiseTearout() {
+                            var myDropTarget = tearElement.parentNode,
+                                offset = { x: 0, y: 0 },
+                                currentlyDragging = false;
+
+                            var me = {};
```

Comment:
Yes, but in general it's a way to structure the code to have a more fluent API to do things like `me.method().method()...` in the mouse handling methods below.